### PR TITLE
Create table for vet clinic database.

### PR DESCRIPTION
**Pull request by Besufekad-HAZ**
# Hello Reviewer! 👋🏾 

## - In this pull request, I did the following things:-
<br />
- Created a database named vet_clinic.
- Created a table named animals with the following columns:

- [ ] - id: integer
- [ ] - name: string
- [ ] - date_of_birth: date
- [ ] - escape_attempts: integer
- [ ] - neutered: boolean
- [ ] - weight_kg: decimal

- Copied the SQL used in the previous point into the schema.sql file in the generated repository.
- Inserted the following data:

- [ ] - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
- [ ] - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
- [ ] - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
- [ ] - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.

- Copied the SQL used in the previous point into the data.sql file in the generated repository.
- Wrote queries for the following quests:

- [ ] - Find all animals whose name ends in "mon".
- [ ] - List the name of all animals born between 2016 and 2019.
- [ ] - List the name of all animals that are neutered and have less than 3 escape attempts.
- [ ] - List the date of birth of all animals named either "Agumon" or "Pikachu".
- [ ] - List name and escape attempts of animals that weigh more than 10.5kg.
- [ ] - Find all animals that are neutered.
- [ ] - Find all animals not named Gabumon.
- [ ] - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg).

- Copied the SQL used in the previous point into the queries.sql file in the generated repository.
- Added a screenshot below with the results of the queries to the Pull Request description.
                             <br />
image
                            <br />
image
                             <br />
image
                             <br />
image
                              <br />
image
                              <br />
image
                               <br />
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,27 +215,23 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com:Besufekad-HAZ/Vet_Clinic.git/issues).
```
Comment (1GORDON) [inline README.md]: Great job @Besufekad-HAZ 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Besufekad-HAZ ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] I would suggest that each query has a unique screenshot for easy identification. This will be easier for the reviewer to see.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Besufekad-HAZ, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Besufekad-HAZ** [comment]
Thanks for the professional review.

**Pull request merged**